### ISSUE TITLE
Z25 shall a reference to ilijasrb's custom part be added?

### ISSUE THREAD

**Issue by mi-hol** [Engenia/Engenia#10]
http://www.engenia.com.au/10_Diagnostics/03_Engine/HotStart.html
Add in Paragraph "The  Solution" link to https://www.benelliforum.com/threads/z25-upgraded-new-shafts.33385/#post-271454  ?

**Comment by Engenia** [comment]
Good idea. Need to add Ilija's contact details though - ILIJA Zivanovic <motoradic@gmail.com>

Ilija has done some good work for us, including an alternative stepper (Part number is 9945635,originally from Fiat Marea 1600 16v). The valve needs to be changed but otherwise it works well.

**Comment by mi-hol** [comment]
Will you verify if he is ok with this or shall I?

**Comment by mi-hol** [comment]
ANd btw isn't "P/N: R310032007000 "Washer, Special" (Belville washer)" just a standard DIN 137 A washer made from spring steel?

**Comment by Engenia** [comment]
Please go ahead and contact Ilija.

A DIN 137 Form A washer is also known as a crinkle or wave washer. The load is supported at only 4 points
A Bellville washer is a cone shape in which the load is supported by the inner and outer circumferences.

**Comment by Engenia** [comment]
Ilija was happy for the reference btw.

**Comment by mi-hol** [comment]
Yeah, have this on my ToDo list, but resolving the [pending .md conversation](https://github.com/Engenia/Engenia/pull/8#issuecomment-1890433621) is the first prio now.
Assuming we can get it resolved, changes to pages will be much simpler

**Comment by mi-hol** [comment]
> A DIN 137 Form A washer is also known as a crinkle or wave washer

right, I was wrong and would now refer to a DIN 2093 made of spring steel instead.
Does that make sense?
